dask_sql/physical/rel/custom/create_model.py

```diff
@@ -183,7 +184,13 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
 
             delayed_model = [delayed(model.fit)(x_p, y_p) for x_p, y_p in zip(X_d, y_d)]
             model = delayed_model[0].compute()
-            model = ParallelPostFit(estimator=model)
+            output_meta = np.array([])
```

Comment:
Vibhu's comment: I don't think we can just hardcode the meta to be `output_meta` to be `np.array([])`. We also use cuML for this case and that outputs a cuDF Series.